.circleci/config.yml

```diff
@@ -738,11 +800,26 @@ jobs:
               echo "No report zip files found, skipping blob merge."
             fi
       - store_artifacts:
-          path: blob-report
+          path: artifacts/blob-report
       - run:
           name: Merge html Reports
           command: |
             cd artifacts/blob-report
+            ls -l
+            JOB_ID=$(cat functional_test_job_id.txt)
+            echo "Fetching reports from circle ci artifact storarge for $JOB_ID"
+            function fetch_report() {
+              echo "Fetching reports-$1.zip"
+              wget https://output.circle-artifacts.com/output/job/${JOB_ID}/artifacts/$1/artifacts/blob-report/reports-$1.zip
```

Comment:
This is doing the same thing as the `fetch_report` function in the `Merge blob reports` step. Is this right?

Reply:
Yes that's correct. I made the changes that will fetch only once and then create a merged blob report and an HTML report.